doc/getting_started.mld

```diff
@@ -0,0 +1,78 @@
+{0 OCamlFormat Manual}
+
+{1 Getting started}
+
+{2 Installation}
+
+OCamlFormat can be installed with `opam`:
+
+{[
+opam install ocamlformat
+]}
+
+Alternatively, it can be built manually with:
+
+{[
+dune subst
+dune build @all
+]}
+
+{2 Formatting code!}
+
+To manually invoke OCamlformat the general command is:
+
+{[
+ocamlformat [OPTION]... [SRC]...
+]}
+
+The most common usecase involves using the Dune build system, once your project is correctly setup (see {{:https://dune.readthedocs.io/en/stable/formatting.html#formatting-a-project}Dune's manual}) you can reformat your project using:
+
+{[
+dune build @fmt
+]}
+
+{2 Code style}
+
+There are a number of preset code style profiles, selected using the `--profile` option by passing `--profile=<name>` on the command line or adding `profile = <name>` to an .ocamlformat configuration file. Each profile is a collection of settings for all options, overriding lower priority configuration of individual options. So a profile can be selected and then individual options can be overridden if desired.
```

Comment:
We should maybe add a list of profiles somewhere

Reply:
done